Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -259,7 +259,7 @@ def _process_beam(self, ws):
             att_coeff = run.getLogData('attenuator.attenuation_coefficient').value
         elif run.hasProperty('attenuator.attenuation_value'):
             att_value = run.getLogData('attenuator.attenuation_value').value
-            if att_value < 10 and self._instrument == 'D33':
+            if int(att_value) < 10 and self._instrument == 'D33':
```

Comment:
are `att_value` only ever strings of ints? `int("10.0")` doesn't work, for example

Reply:
Good question, actually it's a numeric log, I don't know why python3 takes a string as the value.

Reply:
The idea is that if it is less than 10, it means it's the index of the attenuator, not the value.